peripherals/README.md

```diff
@@ -0,0 +1,77 @@
+# Quick start using feagi_agent
+1) `git clone https://github.com/feagi/feagi.git`
+2) `cd feagi/docker`
+3) `docker compose -f feagi.yml build`
+4) Wait until #3 step is complete.
+5) `docker compose -f feagi.yml up`
+6) load it in your preferred browser: `http://localhost:3000/genome/mode`
```

Comment:
the path /genome/mode could well change in future. Lets point to the home page of FEAGI and guide them in a general manner to get to the genome page.